main.py

```
@@ -10,6 +10,22 @@
 app = FastAPI()
 from app.api.endpoints.chat import router as chat_router
 
+# Exception Handlers
+from fastapi.exceptions import RequestValidationError
```

Ruff (0.8.2)

Module level import not at top of file

(E402)